ledger-tool/src/main.rs

```diff
@@ -2045,9 +2070,14 @@ fn main() {
         ("purge", Some(arg_matches)) => {
             let start_slot = value_t_or_exit!(arg_matches, "start_slot", Slot);
             let end_slot = value_t!(arg_matches, "end_slot", Slot).ok();
-            let no_compaction = arg_matches.is_present("no-compaction");
```

Comment:
"no-compaction" should be "no_compaction": https://github.com/solana-labs/solana/pull/11052/files#r509242376